src/views/unauthenticated/NewUserHomeInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { SignInButton } from "../../api";
+
+export function NewUserHomeInfo() {
+	const navigate = useNavigate();
+
+	return (
+		<>
+			<h1>Welcome to Smart shopping list</h1>
+			<p>
+				The next best thing to having someone else do the shopping for you!
+				Create and manage smart lists learn your habits to let you know exactly
+				when you will need t
```

Comment:
typo?

Reply:
Definetly a typo 😅 not even sure where the rest of the sentence got loss at. 😂